cim-forward-route/src/main/java/com/crossoverjie/cim/route/config/BeanConfig.java

```diff
@@ -109,6 +114,27 @@ public RouteHandle buildRouteHandle() throws Exception {
 
     }
 
+    @Bean
+    public LoadingCache<Long, CIMUserInfo> USER_INFO_MAP(RedisTemplate<String, String> redisTemplate) {
+        return CacheBuilder.newBuilder()
+                .initialCapacity(64)
+                .maximumSize(1024)
+                .concurrencyLevel(Runtime.getRuntime().availableProcessors())
+                .expireAfterWrite(10, TimeUnit.MINUTES)
+                .build(new CacheLoader<Long, CIMUserInfo>() {
+                    @Override
+                    public CIMUserInfo load(Long userId) throws Exception {
+                        CIMUserInfo cimUserInfo = null;
+                        String sendUserName = redisTemplate.opsForValue().get(ACCOUNT_PREFIX + userId);
+                        if (sendUserName == null) {
+                            sendUserName = "unLoginUser";
```

Comment:
```suggestion
                            return null;
```

When the `sendUserName`  empty, null can be returned directly because `Optional` is used in the calling place.